static_src/components/users_invite.jsx

```diff
@@ -16,21 +16,31 @@ import { validateString } from '../util/validators';
 const USERS_INVITE_FORM_GUID = 'users-invite-form';
 
 const propTypes = {
+  inviteDisabled: React.PropTypes.bool,
+  currentUserAccess: React.PropTypes.bool,
   error: React.PropTypes.object
 };
 const defaultProps = {};
 
+function stateSetter(props) {
```

Comment:
An example component that doesn't have state at all, and is controlled through props: https://github.com/18F/cg-dashboard/blob/master/static_src/components/org_quicklook.jsx.